### DISCUSSION TITLE
How to Check If the battery is replaced by a Third party repairshop or not?

### DISCUSSION THREAD

**Discussion by dipcse07** [YueChen-C/py-ios-device#116] [Q&A]
These days I have been using pyidevice battery extremely. But after all the search and going through the codes i do not find if there is an option for checking if the battery is **replaced or not**. I am attaching the screen shots of such two iphones in which the battery was replaced !! and in the **battery Health** its showing **Unknown Part** in one and in another it shows **Service**

_**So My Question is: Is there any way to get this info via pyidevice?**_
@YueChen-C 

Thanks Again man.. Your works helping me a lot to learn.. !!

![IMG_2103_r](https://github.com/YueChen-C/py-ios-device/assets/19185636/e8b0e551-e2e4-4894-acef-e07d5150b9fc)![IMG_0001 2 (1)_r](https://github.com/YueChen-C/py-ios-device/assets/19185636/8f5faf6c-90c6-4d0a-8d10-e902352da986)

**Comment by dipcse07** [comment]
@KazuCocoa @YueChen-C @tomriddly  can anyone help me out with this or any advice for this please?

**Comment by YueChen-C** [comment]
I checked the diagnostic service, but found no relevant information

**Reply by dipcse07** [reply]
@YueChen-C is there any other way to get this info or determine whether the battery is replaced or not?
